### PR TITLE
Sexy tits: the list on the front page

### PR DESCRIPTION
**Pull request by behaviary**
## Article ranking

Just a PR to explore the way that the front page should initially list articles before we get the votes going full force. 
### tl;dr

There are going to be some interesting problems with this as it aggrades (I googled antonym of degrade and got that; it's badass) from 0 to n users. The way that articles get listed is eventually going to have to be pretty graceful to show the most interesting stuff. 
### Problems and ideas:
- There is not yet a coherent way to show useful articles before some unknown critical mass of votes is reached.
- If votes are given full weight over date, then the low-vote world of the first few articles is going to be pretty ugly.
- If date is given full weight, then it will only ever look like reddit new.
- There is some quantitative goldilox area between votes and recentness of an article.
### Early solutions:
- In the initial version we'll have to have a simple hybrid between most votes and most recently posted article for the list of things to be valuable right out of the gate OR we will have to populate it. 
- If we go with a hybrid, votes should decrement over time and weight should be given to createdAt/updatedAt to determine rank.

Thoughts?

### CODE REVIEW

**Comment by behaviary** [comment]
Another note on this PR: I left some stuff in the view that is helpful for keeping track of the ids and dates. That's all the weird shit added to the templates.

**Comment by behaviary** [comment]
I smiled at this.

**Comment by behaviary** [comment]
Also, agree on the search magic and links. Something I'll start getting the
old implementation engine chugging on when I kick around.

The front page can totally wait till the actual articles are a little
hashed out. This is a stretch, but I think it would be really cool if
subjects come out of a basic ML algo. If it's doable that is. I've been
looking into it a bit and I think it is. More on that when I have more on
it though.
